apinotes/CMakeLists.txt

```diff
@@ -28,10 +28,6 @@ add_custom_target("copy_apinotes"
     COMMENT "Copying API notes to ${output_dir}"
     SOURCES "${sources}")
 
-# This is treated as an OPTIONAL target because if we don't build the SDK
-# overlay, the files will be missing anyway. It also allows us to build
-# single overlays without installing the API notes.
 swift_install_in_component(DIRECTORY "${output_dir}"
                            DESTINATION "lib/swift/"
-                           COMPONENT sdk-overlay
-                           OPTIONAL)
+                           COMPONENT compiler)
```

Comment:
As long as they _don't_ get installed on Linux…

Reply:
Maybe we should only make this target if building for an Apple platform.

Reply:
It's tricky because of cross-compilers, but that might be a case we can afford to ignore for now.